app/services/sufia/admin/depositor_stats.rb

```diff
@@ -0,0 +1,46 @@
+# Gather information about the depositors who have contributed to the repository
+class Sufia::Admin::DepositorStats
+  include Blacklight::SearchHelper
+
+  # @param [String] start_date optionally specify the start date to gather the stats from
+  # @param [String] end_date optionally specify the end date to gather the stats from
+  def initialize(start_date, end_date)
+    @start_dt = Time.zone.parse(start_date) unless start_date.blank?
+    @end_dt = Time.zone.parse(end_date).end_of_day unless end_date.blank?
+  end
+
+  attr_accessor :start_dt, :end_dt
+
+  def depositors
+    [].tap do |depositors|
```

Comment:
I know this is just copying from above.. but can't this be simplified? `results.each_slice(2).map`, if nothing else?